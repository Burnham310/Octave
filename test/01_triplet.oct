main = Section(Pitch) ;; {
    1 1 5 5 6 6 5'/6 6'/6 5'/6 4 4 3 3 2 2 1'/6 1'2/6
}
