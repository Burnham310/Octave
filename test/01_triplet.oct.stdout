[
{
  "content": 60,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 60,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 67,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 67,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 69,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 69,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 67,
  "duration": 0.3333333432674408,
  "gap": 0.3333333432674408,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 69,
  "duration": 0.3333333432674408,
  "gap": 0.3333333432674408,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 67,
  "duration": 0.3333333432674408,
  "gap": 0.3333333432674408,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 65,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 65,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 64,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 64,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 62,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 62,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 60,
  "duration": 0.3333333432674408,
  "gap": 0.3333333432674408,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 60,
  "duration": 0.6666666865348816,
  "gap": 0.6666666865348816,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "instrument": "dummy"
}

]